src/python/benchUtil.py

```diff
@@ -1279,6 +1332,19 @@ def simpleReport(self, baseLogFiles, cmpLogFiles, jira=False, html=False, baseDe
                         
     w = writer
 
+    catSet = set()
+
+    baseLatencyMetrics = computeTaskLatencies(baseTaskLatencies, catSet)
+    cmpLatencyMetrics = computeTaskLatencies(cmpTaskLatencies, catSet)
+
+    for currentCat in catSet:
+      currentBaseMetrics = baseLatencyMetrics[currentCat] 
+      currentCmpMetrics = cmpLatencyMetrics[currentCat]
+      pctP50 = 100*(currentCmpMetrics['p50'] - currentBaseMetrics['p50'])/currentBaseMetrics['p50']
+      pctP90 = 100*(currentCmpMetrics['p90'] - currentBaseMetrics['p90'])/currentBaseMetrics['p90']
+      print ('||Task %s||P50 Base %s||P50 Cmp %s||Pct Diff %s||P90 Base %s||P90 Cmp %s||Pct Diff %s' %
```

Comment:
Can we print P99 and P99.9 here too?

Reply:
Apologies, should have been clearer here.

I did not add it to the default printing because then we have too many columns which messes up formatting. I am happy to add them as is, or remove P0 or P50 from the printing stack.

Reply:
I think it's OK to include all columns -- these render well in Jira.